CHANGELOG.md

```diff
@@ -37,7 +37,8 @@ update the Unreleased link so that it compares against the latest release tag.
 ### Fixed
 
  - Equalizer now reports the correct equation and values used to perform the adjustment. [#3987](https://github.com/USGS-Astrogeology/ISIS3/issues/3987)
- 
+ - Map2cam now uses virtual bands when swithing bands to enable cube attributes. [#3856](https://github.com/USGS-Astrogeology/ISIS3/issues/3856)
```

Comment:
This reads as being for developers instead of users. Maybe something about map2cam working correctly when specifying match cube bands on the command line.